src/BlazorUI/Bit.BlazorUI/Components/Surfaces/ScrollablePane/BitScrollablePane.ts

```
@@ -0,0 +1,9 @@
+﻿namespace BitBlazorUI {
+    export class ScrollablePane {
+        public static scrollToEnd(element: HTMLInputElement) {
+            if (!element) return;
+
+            element.scrollTo(element.scrollWidth, element.scrollHeight);
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)